src/app/loginComponents/onboarding/downloadcliclient/downloadcliclient.component.ts

```diff
@@ -63,10 +64,10 @@ Setup our Dockstore CLI application to start launching workflows from the comman
 
 #### Requirements
 1. Linux/Ubuntu (Recommended - Tested on 18.04.3 LTS) or Mac OS X machine
-2. Java 11 (Tested with OpenJDK 11, Oracle JDK may work but is untested)
+2. Java 11 (Tested with OpenJDK 11 and Oracle JDK)
```

Comment:
This isn't right, we do not test with Oracle JDK. (at least not in the sense of unit/integration tests)

Reply:
Changed back